service/security/test/src/test/resources/features/GroupServiceUnitTests.feature

```diff
@@ -0,0 +1,387 @@
+    And No exception was thrown
+
+  Scenario: Editing Access Group description to non-unique one
+  Create two Access Groups with different valid names and different description. Edit description of one group to the same description of the second group.
+  Kapua should not return any errors.
+    Given I create the group with name "AccessGroup" and description "description"
+    And I create the group with name "NewAccessGroup" and description "description2"
+    Then I update the group description to "description"
+    Then No exception was thrown
+
+  Scenario: Editing Access Group description to description with special symbols
+  Create unique Access Group with description. Edit the group description to contain valid special symbols, enter all possible characters.
+  Kapua should not return any errors.
+    Given I create the group with name "NewAccessGroup" and description "description"
+    When I update the group description to "!description@#$%%"
+    Then The group was correctly updated
+    And No exception was thrown
+
+  Scenario: Editing Access Group description to description with numbers
+  Create unique Access Group with description. Edit the group description to contain numbers.
+  Kapua should not return any errors.
+    Given I create the group with name "NewAccessGroup" and description "description"
+    When I update the group description to "description123"
+    Then The group was correctly updated
+    And No exception was thrown
+
+  Scenario: Editing Access Group description to description with only numbers
+  Create unique Access Group with description. Edit the group description to contain only numbers.
+  Kapua should not return any errors.
+    Given I create the group with name "NewAccessGroup" and description "description"
+    When I update the group description to "12345"
+    Then The group was correctly updated
+    And No exception was thrown
+
+  Scenario: Editing Access Group description to short description
+  Create unique Access Group with description. Edit the group description to short description, description that contains 3 characters.
+  Kapua should not return any errors.
+    Given I create the group with name "NewAccessGroup" and description "description"
+    When I update the group description to "a"
+    Then The group was correctly updated
+    And No exception was thrown
+
+  Scenario: Editing Access Group description to long description
+  Create unique Access Group with description. Edit the group description to long description, description that contains 255 characters.
+  Kapua should not return any errors.
+    Given I create the group with name "NewAccessGroup" and description "description"
+    When I update the group description to "NewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNA"
```

Comment:
Please change this step to: "When I update description of group "newAccessGroup" from "description" to "NewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNewAccessGroupNA"" so the implementation receives two parameters instead of one. This way the step is more clear and easier to understand.